src/command.rs

```diff
@@ -73,6 +80,22 @@ impl Command {
         }
         state.loop_counter += 1;
       }
+      Self::Open(path) => {
+        process::Command::new(
+          env::var("DEGENERATE_OPEN_COMMAND").unwrap_or(
+            {if cfg!(target_os = "macos") {
+              Ok("open")
+            } else if cfg!(target_os = "linux") {
+              Ok("xdg-open")
+            } else {
+              Err("Please supply an open command by setting the `DEGENERATE_OPEN_COMMAND` environment variable")
+            }
+            }?.to_string()
+          )
```

Comment:
The use of `Ok` and `Err` like this is a little weird. We can just return the error case, I think:

```suggestion
          env::var("DEGENERATE_OPEN_COMMAND").unwrap_or(
            {if cfg!(target_os = "macos") {
              "open".to_string()
            } else if cfg!(target_os = "linux") {
              "xdg-open".to_string()
            } else {
              return Err("Please supply an open command by setting the `DEGENERATE_OPEN_COMMAND` environment variable".into());
            }
            }
          )
```